pkg/conversions/pod_conversion_test.go

```diff
@@ -0,0 +1,102 @@
+package conversions
+
+import (
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+	corev1 "k8s.io/api/core/v1"
+	v1 "k8s.io/api/core/v1"
+	metaV1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+)
+
+func TestConvertObj(t *testing.T) {
+	podConverter := NewPodConverter()
+
+	annotations := make(map[string]string)
+	annotations["random"] = "TestValue"
+	annotations["appmesh.k8s.aws/cpuLimit"] = "60"
+
+	labels := make(map[string]string)
+	labels["app"] = "TestApp"
+	labels["role"] = "front"
+
+	commands := []string{"sh", "-c", "echo Container 1 is Running; sleep 360000"}
+
+	container := corev1.Container{
+		Name:            "busybox",
+		Image:           "busybox",
+		ImagePullPolicy: "IfNotPresent",
+		Command:         commands,
+	}
+
+	containers := make([]corev1.Container, 0)
+	containers = append(containers, container)
+
+	oldPod := &corev1.Pod{
+		ObjectMeta: metaV1.ObjectMeta{
+			Name:        "TestPod",
+			Namespace:   "TestNameSpace",
+			Annotations: annotations,
+			Labels:      labels,
+		},
+		Spec: v1.PodSpec{
+			NodeName:   "TestNode",
+			Containers: containers,
+		},
+	}
+
+	convertedObj, err := podConverter.ConvertObject(oldPod)
+	if err != nil {
+		t.Error(err)
```

Comment:
Can you check error assert statements, they may be better for checking if there is no error.
https://godoc.org/github.com/stretchr/testify/assert